packages/functions/package.json

```diff
@@ -7,7 +7,27 @@
     "shell": "npm run build && firebase functions:shell",
     "start": "npm run shell",
     "deploy": "firebase deploy --only functions",
-    "logs": "firebase functions:log"
+    "logs": "firebase functions:log",
+    "deploy-function-dev": "func() { shx cp -rf ../accounts/service-account-dev ./src/service-account.ts && firebase use pipeline-game-dev && npm run _clean && npm --prefix ./ run lint && npm --prefix ./ run build && firebase deploy --only functions:\"$1\" && npm run _restore-fake-account; }; func",
```

Comment:
On Windows this npm script does not work --> " 'func' is not recognized as an internal or external command, operable program or batch file."